src/ros_bridge/src/webots_controller.cpp

```diff
@@ -46,6 +46,14 @@ class WebotsController : public rclcpp::Node {
     // Client construction and connecting
     this->get_parameter("host", host_);
     this->get_parameter("port", port_);
+
+    const char* env_p = std::getenv("ROBOCUP_SIMULATOR_ADDR");
```

Comment:
I guess this works but as this Docker image is supposed to be an example for other teams, I personally would prefer to do it in a clean way, i.e. to write an entrypoint.sh and then pass all env vars via CLI arguments to the executable.